src/ansible_creator/resources/collection_project/plugins/action/hello_world.py.j2

```diff
@@ -0,0 +1,89 @@
+{# action_plugin_template.j2 #}
+{%- set action_name = plugin_name | default("hello_world") -%}
+{%- set author = author | default("Your Name") -%}
+{%- set description = description | default("A custom action plugin for Ansible.") -%}
+{%- set license = license | default("GPL-3.0-or-later") -%}
+# {{ action_name }}.py - {{ description }}
+# Author: {{ author }}
+# License: {{ license }}
+# pylint: disable=E0401
+
+from __future__ import absolute_import, annotations, division, print_function
+
+__metaclass__ = type  # pylint: disable=C0103
+
+from typing import TYPE_CHECKING
+from ansible.plugins.action import ActionBase  # type: ignore
+
+
+if TYPE_CHECKING:
+    from typing import Optional, Dict, Any
+
+
+DOCUMENTATION = """
```

Comment:
doc can be removed from the action plugin file, ansible will use the module file fro these